src/main/java/Deadline.java

```diff
@@ -0,0 +1,15 @@
+
+public class Deadline extends Task {
+
+    private String by;
```

Comment:
'by' is a bit confusing, perhaps consider something more specific like 'dueDate' for clarity so it's easier to understand what this variable is for